sessionctx/variable/tidb_vars.go

```diff
@@ -334,7 +334,8 @@ const (
 	DefTiDBUseFastAnalyze              = false
 	DefTiDBSkipIsolationLevelCheck     = false
 	DefTiDBWaitTableSplitFinish        = false
-	DefTiDBExpensiveQueryTimeThreshold = 60 // 60s
+	DefTiDBExpensiveQueryTimeThreshold = 100 // 100 ms
```

Comment:
100ms is too small as the default value for `ExpensiveQueryTimeThreshold`, it makes TiDB print a log of logs.
I think 60s may be a better default value here.

And note that the check interval of the ticker is not necessary to be `ExpensiveQueryTimeThreshold / 2` because the check interval value is shared with `MAX_EXECUTION_TIME` check interval, maybe 100ms is a good choice.

It means the `MAX_EXECUTION_TIME` will be less accurate if the check interval is 100ms.  That's acceptable as the code is simple enough in my opinion. We can polish the code after this feature is merged, adopting a dynamic check interval.